tests/operator_test.go

```diff
@@ -585,20 +570,12 @@ spec:
 	AfterEach(func() {
 		ignoreDeleteOriginalKV := true
 
-		curKubeVirtConfig, err := virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Get("kubevirt-config", metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-
 		// if revision changed, patch data and reload everything
-		if curKubeVirtConfig.ResourceVersion != originalKubeVirtConfig.ResourceVersion {
+		if !reflect.DeepEqual(originalKubeVirtConfig, currentConfiguration) {
```

Comment:
why would this ever not be equal?  current is never updated after being set to original.